core/src/main/res/values-ar/strings.xml

```diff
@@ -129,23 +116,58 @@
   <string name="pref_external_link_popup_summary">عرض المنبثق للتحذير من التكاليف الإضافية أو عدم العمل في الروابط بلا اتصال.</string>
   <string name="external_link_popup_dialog_title">دخول الرابط الخارجي</string>
   <string name="external_link_popup_dialog_message">أنت تقوم بإدخال رابط خارجي، قد يؤدي ذلك إلى تكاليف إضافية لنقل البيانات أو لن يعمل فقط عندما تكون غير متصل بالإنترنت، هل تريد الاستمرار؟</string>
   <string name="do_not_ask_anymore">لا تسأل بعد الآن</string>
   <string name="your_languages">اللغات المختارة:</string>
   <string name="other_languages">لغات أخرى:</string>
-
+  <string name="no_items_msg">لا توجد عناصر متاحة</string>
+  <string name="ongoing_download_channel_name">التنزيلات الجارية</string>
+  <string name="ongoing_download_channel_desc">تصنيف لتنزيلات ملف zim المستمرة.</string>
+  <string name="crash_title">حسنا، هذا أمر محرج</string>
+  <string name="crash_description">يبدو أننا انهرنا.\n\nهل تمانع في مساعدتنا في حل هذه المشكلة عن طريق إرسال المعلومات التالية؟</string>
+  <string name="crash_checkbox_language">إعدادات لغتك</string>
+  <string name="crash_checkbox_zimfiles">قائمة ملفات Zim الخاصة بك</string>
+  <string name="crash_checkbox_exception">تفاصيل الانهيار</string>
+  <string name="crash_checkbox_logs">سجلات التطبيق</string>
+  <string name="crash_checkbox_device">تفاصيل الجهاز</string>
+  <string name="crash_button_confirm">إرسال التفاصيل</string>
+  <string name="crash_button_decline">لا شكرا</string>
+  <string name="shortcut_disabled_message">اختصار غير متوفر</string>
+  <string name="new_tab_shortcut_label">علامة تبويب جديدة</string>
+  <string name="get_content_shortcut_label">الحصول على المحتوى</string>
+  <string name="pic">صورة</string>
+  <string name="vid">فيد</string>
+  <string name="articleCount">%s مقالات</string>
+  <string name="get_started">ابدأ</string>
+  <string name="download_books">تنزيل الكتب</string>
+  <string name="human_kind_knowledge">المعرفة الإنسانية، على هاتفك.</string>
+  <string name="welcome_to_the_family">مرحبا بك في العائلة</string>
+  <string name="save_books_offline">حفظ الكتب دون اتصال</string>
+  <string name="download_books_message">قم بتنزيل الكتب واقرأ أينما كنت.</string>
+  <string name="unable_to_add_to_bookmarks">غير قادر على الإضافة إلى الإشارات المرجعية</string>
+  <string name="go_to_previous_page">الانتقال إلى الصفحة السابقة</string>
+  <string name="go_to_next_page">انتقل إلى الصفحة التالية</string>
+  <string name="table_of_contents">جدول المحتويات</string>
+  <string name="select_languages">اختر اللغات</string>
+  <string name="save_languages">حفظ اللغات</string>
+  <string name="send_feedback">إرسال تعليق</string>
+  <string name="expand">وسع</string>
+  <string name="history">تاريخ</string>
+  <string name="history_from_current_book">التاريخ من الكتاب الحالي</string>
+  <string name="search_history">سجل البحث</string>
+  <string name="selected_items">تم تحديد %1$d</string>
```

Comment:
I would say this is wrong
Correct example 
```
<string name="welcome_messages">Hello, %1$s! You have %2$d new messages.</string>
```

Reply:
I'm not so sure about the regex to match the %... stuff? Do you know what it should be?

Reply:
```
// copied from java.util.Formatter
// %[argument_index$][flags][width][.precision][t]conversion
private static final String formatSpecifier
    = "%(\\d+\\$)?([-#+ 0,(\\<]*)?(\\d+)?(\\.\\d+)?([tT])?([a-zA-Z%])";
```